scripts/template/documentation.md.jst

```diff
@@ -0,0 +1,19 @@
+# <%= description %>
+
+<!-- More detailed description -->
+<% if (fixable === true) { %>
```

Comment:
```suggestion
<% if (isFixable === 'true') { %>
```

Reply:
It's boolean, and it's named fixable in create-rule.js

Reply:
The quoting was a mistake. The point was the naming.